components/dialog/test/dialog-fullscreen.test.js

```diff
@@ -0,0 +1,14 @@
+import '../dialog.js';
```

Comment:
Should this be importing `dialog-fullscreen.js` and `runConstructor('d2l-dialog-fullscreen')`?

Reply:
Yikes, yes.  Thanks.  Too much copy pasty.